### PR TITLE
ZEN-20018 Run zeneventserver-create-db in upgrade.sh

### PR DESCRIPTION
**Pull request by ed-**

### CODE REVIEW

File: bin/zenrun.d/upgrade.sh [hunk 1]
```diff
@@ -17,6 +17,16 @@ doUpgrade() {
...
+    zengc=$ZENHOME/bin/zenglobalconf
+    dbtype=$(${zengc} -p zodb-db-type)
+    host=$(${zengc} -p zodb-host)
+    port=$(${zengc} -p zodb-port)
+    user=$(${zengc} -p zodb-user)
+    userpass=$(${zengc} -p zodb-password)
+    /opt/zenoss/bin/zeneventserver-create-db \
+        --dbtype $dbtype --dbhost $host --dbport $port \
+        --dbuser $user --dbpass "${userpass}" \
+        --update_schema_only
```
Comment (alapidas) [inline bin/zenrun.d/upgrade.sh]: Can you please `|| return "$?"` here as well?

**Comment by alapidas** [comment]
I _think_ we want to maybe run this for all databases, not just zep

**Comment by alapidas** [comment]
Does `zodb_session` actually exist in 5.1?  I think @jplouis replaced that with a memcache-based solution

**Comment by ed-** [comment]
It was in my develop environment; scratching and rebuilding to get a dead reckoning.

**Pull request merged**